chatroom.py

```diff
@@ -1,6 +1,16 @@
 import random
+from os import system, name
+from colorama import init, Fore
```

Comment:
To get the colour stuff working, you need to install the colorama module using:
python - m pip install colorama